docs/platform-openrpc.json

```diff
@@ -0,0 +1,1208 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "Ledger Live Platform API",
+        "version": "1.0.0",
+        "description": "An JSON-RPC 2.0 API between Ledger Live and third-party applications",
+        "license": {
+            "name": "MIT",
+            "url": "https://opensource.org/licenses/MIT"
+        }
+    },
+    "methods": [
+        {
+            "name": "ledgerlive.open_scheme",
```

Comment:
Do we have a list of schemes that can be opened from an iframe documented somewhere?

Reply:
I don't think it's documented, but yep I would be interested on some documentation about this. 
@gre Do you  have some inputs ?